src/test/suite/fixtures/recorded/languages/rust/chuckNameJury.yml

```diff
@@ -0,0 +1,36 @@
+languageId: rust
+command:
+  spokenForm: chuck name jury
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: j}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: name}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    match user {
+        User { first_name: "John", last_name: "Doe" } => {},
+    }
+  selections:
+    - anchor: {line: 3, character: 0}
+      active: {line: 3, character: 0}
+  marks:
+    default.j:
+      start: {line: 1, character: 24}
+      end: {line: 1, character: 28}
+finalState:
+  documentContents: |
+    match user {
+        User { : "John", last_name: "Doe" } => {},
```

Comment:
Is there a similar shorthand supported here?  Eg

```rust
match user {
        User { first_name, last_name: "Doe" } => {},
}
```

If so I'd remove leading / trailing `:` on "chuck"

Reply:
That would be a name binding, but I think that'd make sense.

Reply:
@purpleP how come this one also has leading `:` still?